applications/MeshingApplication/custom_processes/mmg_process.cpp

```diff
@@ -760,7 +760,7 @@ void MmgProcess<TDim>::ExecuteRemeshing()
     /* Unmoving the original mesh to be able to interpolate */
     if (mFramework == Lagrangian) 
     {
-        NodesArrayType& old_nodes_array = old_nodes_array.Nodes();
+        NodesArrayType& old_nodes_array = r_old_model_part.Nodes();
```

Comment:
take care. I think that here u are accessing the underlying container.

you should do

         ModelPart::NodesContainerType& old_nodes_array ...

or
    
         auto& old_nodes_array ...

Reply:
I use 
~~~
typedef ModelPart::NodesContainerType                        NodesArrayType;
~~~
so is OK